paper_framework_sources/p2079_system_execution_context.bs

```diff
@@ -372,10 +353,11 @@ public:
 };
 </pre>
 
- - `get_system_scheduler()` returns a scheduler that provides a view on some underlying execution context supporting *parallel forward progress*, with at least one thread of execution (which may be the main thread).
- - two objects returned by `get_system_scheduler()` may share the same execution context.
+ - `get_parallel_scheduler()` returns a scheduler that provides a view on some underlying execution context supporting *parallel forward progress*, with at least one thread of execution (which may be the main thread).
+ - two objects returned by `get_parallel_scheduler()` most frequently share the same execution context.
```

Comment:
I don't really like this wording. What about:
> two objects returned by `get_parallel_scheduler()` will share the same execution context where possible.

Same get out clause, slightly more instructional.